appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Where $ADDRESSBOOKNAME is a name you like to identify the virtual addressbook wi
 
 Given that the configuration is correct, you and your users will be able to search through the LDAP contacts via the global contacts menu. An import action allows to copy the contact over to the best fitting existing addressbook. A redirect takes you the contacts app with the newly created card open.
 ]]></description>
-    <version>1.11.0</version>
+    <version>2.0.0-dev.0</version>
```

Comment:
Should be 1.12

Reply:
SemVersions were agreed. Since the app uses stable branches, 31 is dropped => breaking change => Major.

But you can change it if you want.